agent/app/agent_capability_unix.go

```diff
@@ -232,3 +232,7 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+func defaultIsGuardDutySupported() (bool, error) {
```

Comment:
based on my comment to move pathExists() here, lets add a unit test for this function

Reply:
Now we have the check here and added the unit test.